<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="P4A-508SD66" name="Test1" tests="2" failures="1" timestamp="25 Jul 2018 07:09:20 GMT" time="0.64" errors="0">
  <testcase name="LogInvalid" time="0.02" classname="Framworktests.Logintest">
    <failure type="java.lang.AssertionError" message="expected [Tested] but found [Tested123]">
      <![CDATA[java.lang.AssertionError: expected [Tested] but found [Tested123]
	at Framworktests.Logintest.LogInvalid(Logintest.java:20)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- LogInvalid -->
  <testcase name="LoginValid" time="0.0" classname="Framworktests.Logintest"/>
</testsuite> <!-- Test1 -->
